request.go

```diff
@@ -138,6 +143,12 @@ func (r *Request) Post() *Request {
 	return r
 }
 
+// PostForm ...
+func (r *Request) PostForm() *Request {
+	r.httpVerb = POST
```

Comment:
We can add the header constant here.

Reply:
Fixed

Reply:
Please do add a UT checking if the httpVerb for POST_FORM is POST.

Reply:
fixed